### DISCUSSION TITLE
brew update hangs

### DISCUSSION THREAD

**Discussion by zzhjerry** [Homebrew/discussions#192] [Everyday usage]
Platform: MacOS Catalina V10.15.6
Phenomenon: Run `brew update` and it just hangs without any output information

Output by running `brew update --debug --verbose`, see the gist below:

https://gist.github.com/zzhjerry/c9ad150705a7afbb5dac252efd3ce255

May someone know how this could happen?

**Comment by SMillerDev** [comment]
It looks like it's just slow, that gist looks completely normal
